final_task/setup.py

```diff
@@ -0,0 +1,42 @@
+from setuptools import setup, find_packages
+import os
+
+
+this_directory = os.path.abspath(os.path.dirname(__file__))
+with open(path.join(this_directory, 'README.md'), encoding='utf-8') as fh:
+    long_description = fh.read()
+
+def get_version():
+    basedir = os.path.dirname(__file__)
+    with open(os.path.join(basedir, 'rss_reader/version.py')) as f:
+        VERSION = None
+        exec(f.read())
```

Comment:
Best practice to avoid functions like `exec()` and `eval()` in any situations possible because they are huge hazard to application security. Could you please explain why do you even need it here?

Reply:
Fixed